app/javascript/src/lib/OWLanguageLinter.ts

```diff
@@ -261,7 +263,7 @@ function checkTranslations(content: string): void {
       walk--
     }
 
-    if (parenthesisBeforeAtIndex === -1) {
+    if (!isStatic && parenthesisBeforeAtIndex === -1) {
```

Comment:
We could skip a lot of work here by wrapping the calculation of `parenthesisBeforeAtIndex` and this in an `if (!isStatic)` too